tests/build/build.go

```diff
@@ -123,6 +126,190 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
+					if err != nil {
+						GinkgoWriter.Println("PipelineRun on pull request has not been created yet")
+						return false
+					}
+					pipelineRunPush, err := f.AsKubeAdmin.HasController.GetComponentPipelineRun(annotationsComponentName, applicationName, testNamespace, mergeResult.GetSHA())
+					if err != nil {
+						GinkgoWriter.Println("PipelineRun on push has not been created yet")
+						return false
+					}
+					return pipelineRunPull.HasStarted() && pipelineRunPush.HasStarted()
+				}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRuns to start")
+
+				Expect(f.AsKubeAdmin.TektonController.DeleteAllPipelineRunsInASpecificNamespace(testNamespace)).To(Succeed())
+
+				component, err = f.AsKubeAdmin.HasController.GetHasComponent(annotationsComponentName, testNamespace)
+				Expect(err).ShouldNot(HaveOccurred())
+				Expect(component.Annotations[build.BuildRequestAnnotationName]).To(BeEmpty(), "request annotation was not deleted")
+			})
+
+			It("simple build can be triggered manually in PaC mode", func() {
+				component, err = f.AsKubeAdmin.HasController.GetHasComponent(annotationsComponentName, testNamespace)
+				Expect(err).ShouldNot(HaveOccurred())
+
+				component.Annotations[build.BuildRequestAnnotationName] = build.BuildRequestTriggerSimpleBuildAnnotationValue
+				Expect(f.AsKubeAdmin.CommonController.KubeRest().Update(context.TODO(), component)).To(Succeed())
+
+				Eventually(func() bool {
+					pipelineRun, err := f.AsKubeAdmin.HasController.GetComponentPipelineRun(annotationsComponentName, applicationName, testNamespace, "")
+					if err != nil {
+						GinkgoWriter.Println("PipelineRun has not been created yet")
+						return false
+					}
+					return pipelineRun.HasStarted()
+				}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to start")
+
+				Expect(f.AsKubeAdmin.TektonController.DeleteAllPipelineRunsInASpecificNamespace(testNamespace)).To(Succeed())
+
+				component, err = f.AsKubeAdmin.HasController.GetHasComponent(annotationsComponentName, testNamespace)
+				Expect(err).ShouldNot(HaveOccurred())
+				Expect(component.Annotations[build.BuildRequestAnnotationName]).To(BeEmpty(), "request annotation was not deleted")
+			})
+
+			It("handles invalid request annotation", func() {
+				component, err = f.AsKubeAdmin.HasController.GetHasComponent(annotationsComponentName, testNamespace)
+				Expect(err).ShouldNot(HaveOccurred())
+
+				requestedAction := "foo"
+				component.Annotations[build.BuildRequestAnnotationName] = requestedAction
+				Expect(f.AsKubeAdmin.CommonController.KubeRest().Update(context.TODO(), component)).To(Succeed())
+
+				Consistently(func() bool {
+					pipelineRun, err := f.AsKubeAdmin.HasController.GetComponentPipelineRun(annotationsComponentName, applicationName, testNamespace, "")
+					if err != nil {
+						GinkgoWriter.Println("PipelineRun has not been created yet")
```

Comment:
I don't think this should be logged here, since we're not expecting pipelinerun to get created. I'd reuse [this](https://github.com/redhat-appstudio/e2e-tests/blob/c95caa84cf6a7f33fecbe5dfeb2b30abc47547c7/tests/build/build.go#L496-L500) here instead